website/content/en/guides/advanced/cloudwatch-logs-firehose.md

```diff
@@ -90,12 +86,24 @@ Let's take a look at the configuration we will be using:
   access_key = "${FIREHOSE_ACCESS_KEY}"my secret key" # this will also be set when configuring Firehose
 
 [transforms.cloudwatch]
-  type = "aws_cloudwatch_logs_subscription_parser"
+  type = "remap"
   inputs = ["firehose"]
+  drop_on_error = false
+  source = '''
+    parsed = parse_aws_cloudwatch_log_subscription_message!(.message)
+    . = unnest(parsed.log_events)
+  '''
 
 [transforms.json]
-  type = "json_parser"
+  type = "remap"
   inputs = ["cloudwatch"]
+  drop_on_error = false
+  source = '''
+    log_events = del(.log_events)
+    . |= object!(log_events)
+    message = del(.message)
+    . |= object!(parse_json!(message))
+  '''
```

Comment:
@jszwedko suspects this could be combined using iteration functions, so we'll see how that goes. We'll need to note minimum version in the guide if that's the case.